zbsprimarystorage/ansible/zbsp.py

```
@@ -0,0 +1,188 @@
+else:
+    for pkg in install_rpm_list.split():
+        yum_install_package(pkg, host_post_info)
+    if host_info.major_version >= 7:
+        command = "(which firewalld && service firewalld stop && chkconfig firewalld off) || true"
+        run_remote_command(command, host_post_info)
+set_selinux("state=disabled", host_post_info)
+
+
+# name: make sure MDS has access to storage resources
+command = "[ -f /usr/local/bin/zbsadm ] && /usr/local/bin/zbsadm client deploy --host %s -p %s" % (host_post_info.host, host_post_info.remote_pass)
+run_remote_command(command, host_post_info)
```

Ruff (0.12.2)

`run_remote_command` may be undefined, or defined from star imports

(F405)